cmd/gokr-updater/updater.go

```diff
@@ -3,11 +3,12 @@ package main
 
 import (
 	"flag"
+	"fmt"
+	"github.com/gokrazy/internal/httpclient"
```

Comment:
nit: for consistency, can you please move all github imports to a separate import group (separated by a blank line)?